src/server/rest/v1/service/TagService.ts

```diff
@@ -248,6 +249,17 @@ export default class TagService {
         action: action
       });
     }
+    const nbrTransactions = await TransactionStorage.getTransactions(req.user.tenantID,{ tagIDs:[filteredRequest.id.toUpperCase()] }, Constants.DB_PARAMS_MAX_LIMIT);
```

Comment:
To get the number of Transaction, always use the Constants.DB_PARAMS_COUNT_ONLY otherwise you may have -1 as a result + you are retrieving here all the Transactions just to check the count (rule 22).